src/services/pool/calculator/stable.ts

```diff
@@ -113,7 +114,7 @@ export default class Stable {
     // tokenIndex does not account for pre-minted BPT
     // So you must get the address from an address list that also excludes
     // pre-minted BPT.
-    const tokenOutAddress = this.calc.pool.value.tokensList[tokenIndex];
+    const tokenOutAddress = tokensListExclBpt(this.calc.pool.value)[tokenIndex];
```

Comment:
hope it won't break something

Reply:
This should also be fine, see the comment above.